python/rgbmatrix/core.pyx

```diff
@@ -83,18 +83,100 @@ cdef class FrameCanvas(Canvas):
         def __get__(self): return (<cppinc.FrameCanvas*>self.__getCanvas()).pwmbits()
         def __set__(self, pwmBits): (<cppinc.FrameCanvas*>self.__getCanvas()).SetPWMBits(pwmBits)
 
+cdef class RGBMatrixOptions:
+    def __cinit__(self):
```

Comment:
So if it is hard to call the C++ parsing, maybe we then should do it 'the Python way'. So I don't know what that would be, but I could imagine, that there could be an Init() function for the RGBMatrixOptions that gets a self.parser or something ?
How is it usually done in Python if submodules need to deal with their own options ?

That then can also cover the `--help/-h` case automatically.

Then the implementation should be in RGBMatrixOptions, so it is in one place (and easy to add if there are new options). That way, we don't have them in samplebase.py which is probably not what people would like to use.